components/Hosts.js

```diff
@@ -68,8 +91,8 @@ class Hosts extends React.Component {
             <P textAlign="center">
               <FormattedMessage
                 id="hosts.description"
-                defaultMessage="Fiscal Hosts hold money on behalf of Collectives, taking care of accounting, taxes, invoices, etc. Some also provide extra services. {findOutMoreLink}"
-                values={{ findOutMoreLink }}
+                defaultMessage="{fiscalHostingLink} hold money on behalf of Collectives, taking care of accounting, taxes, invoices, etc. Some also provide extra services. {findOutMoreLink} about {becomingAHostLink}"
```

Comment:
It's error-prone for contributors when strings are broken up like that. Could we do something like:

```jsx
                defaultMessage="<FiscalHostingLink>Fiscal hosts</FiscalHostingLink> hold money on behalf of Collectives, taking care of accounting, taxes, invoices, etc. Some also provide extra services. {findOutMoreLink} about {becomingAHostLink}"
```

Reply:
Corrected all the string in this component. Thanks. 👍🏽